delivery/model.py

```diff
@@ -68,3 +70,43 @@ def from_dict(raw: dict):
                 type_hooks={datetime.date | None: _parse_date_if_present},
             ),
         )
+
+
+@dataclasses.dataclass(frozen=True)
+class ComponentArtefactId:
```

Comment:
ideally, there should also be a util for creating this from cnudie.iter.ResourceNode | SourceName, wdyt?

Reply:
Sure, it _could_ be done, but currently `ComponentArtefactId`  is only instantiated when retrieving data from the delivery-service. Could be useful for comparison between local and retrieved objects, but otherwise I don't see it